### DISCUSSION TITLE
AP mode to WiFi mode

### DISCUSSION THREAD

**Discussion by BramWerbrouck** [RaspAP/raspap-webgui#1610] [Q&A]
Hi, I installed RaspAp correctly and when I start my RPI, the access point is created. I can connect with another device to enter the settings menu. Now I want to go back to WiFi mode, but this does not work. I thought you have to stop the Hotspot with the stop hotspot button, there is already a known network under WiFi client, but then the AP stops but there is no WiFi connection..

**Comment by billz** [comment]
Using Wi-Fi client mode and hosting an AP simultaneously requires a second WLAN adapter. If your goal is to create a wireless repeater, this is described in detail here https://docs.raspap.com/repeater/

**Comment by BramWerbrouck** [comment]
I don't want to use the access point and WiFi simultaneously, I thought that I could use RaspAp to use the RPI as an access point and then
set a WiFi network. Or is there a way to turn of the AP and connect to WiFi (as I said before: turn of the hotspot (Stop Hotspot button in the AP GUI) and connect to a (known) network does not work: the AP stops, but the RPI does not connect to the WiFi 
(the WiFi icon is not visible in the taskbar - next to the clock - I only see 2 red crosses and a tooltip: No wireless interfaces found

**Reply by billz** [reply]
It's not clear what you're trying to achieve here. However, it sounds like you're using a Desktop distro, which is isn't supported by this project (as clearly stated in the docs). Best advice is to review them before proceeding further.

**Reply by BramWerbrouck** [reply]
Isn't this a missed opportunity to also be able to use RaspAp in that way? It could certainly be very useful

**Reply by billz** [reply]
Again, this is covered in the docs https://docs.raspap.com/faq/#distros

**Reply by billz** [reply]
RaspAP already supports a variety of popular Debian Linux-based distros. The exception to this is Desktop, which has been very problematic in the past. If you'd like to see support for this, perhaps you (or someone you know) could contribute to the effort to make this happen.

**Comment by BramWerbrouck** [comment]
Ok, I understand and I am working at it. meanwhile, is there a way to temporarily disable RaspAp so the WifI mode can be used on Raspberry Pi without uninstalling?

**Reply by billz** [reply]
Stop the access point with **Hotspot > Stop hotspot**. Preventing RaspAP from restarting at boot is described here https://docs.raspap.com/faq/#raspap-service

**Reply by BramWerbrouck** [reply]
When I press **Hotstop > Stop hotspot** I still can’t connect to my own WiFi network with my raspberry pi. I got the tooltip (at the place where normally the WiFi sign appears but now shown as 2 little red crosses) **no wireless interfaces found**

**Reply by billz** [reply]
Once again, if you're using desktop this is totally unsupported.

**Reply by BramWerbrouck** [reply]
So you can't just disable the whole RaspAp in de desktop version without uninstalling??

**Reply by billz** [reply]
In this case you should uninstall RaspAP and use a supported distro.

**Reply by frankozland** [reply]
i think im seeing same thing Bill - I turned on ap mode.  Then i tried to go back to wifi - raspap wouldnt let me do that until i edited /etc/raspap/hostapd.ini and removed the uap1 and replaced it with the wifi interface wlan0.

stop everything and then restart and back to normal.
hotel california when turning on WiFi client AP mode?
WifiInterface = wlan0
LogEnable = 1
WifiAPEnable = 0
BridgedEnable = 0
WifiManaged = wlan0

**Reply by billz** [reply]
@frankozland if you engage AP-STA mode, you must disable this option _before_ reverting to a wireless client config.
